### PR TITLE
New Getting Started content for API Testing

### PR DESCRIPTION
**Pull request by lauramthomas**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing.md [hunk 1]
```diff
@@ -1,51 +1,78 @@
...
+</div>
+</div>
+
+<div className="box box4 card">
+<div className="container">
+
+<h2>Testing Private APIs</h2>
+<p></p>
+<p>
+Use Sauce Connect Proxy to test APIs behind a firewall. This launches a secured tunnel between the Sauce Labs API Testing platform and your APIs. See <a href="/secure-connections/sauce-connect">Sauce Connect Proxy</a> for more information.
```
Comment (lysannep) [inline docs/api-testing.md:73]:
```suggestion
Use Sauce Connect Proxy to test APIs behind a firewall. This launches a secure tunnel between the Sauce Labs API Testing platform and your APIs. See <a href="/secure-connections/sauce-connect">Sauce Connect Proxy</a> for more information.
```

File: docs/api-testing.md [hunk 2]
```diff
@@ -1,51 +1,78 @@
...
-title: API Testing Documentation
+title: API Testing
 sidebar_label: Getting Started
-description: Documentation for running API tests with Sauce Labs
+description: Running API tests with Sauce Labs
 hide_table_of_contents: true
 ---
 
-Check out the docs below to get started with API Testing and Monitoring on the Sauce Labs continuous testing cloud.
+At its core, Sauce Labs API Testing allows you to use either and HTTP client or a proprietary test composer to generate and edit intelligent contract or functional tests. The tests can be automated with any CI/CD tool or using the platform’s scheduler. Functional tests can then be used as load tests or scheduled as uptime monitors.
```
Comment (lysannep) [inline docs/api-testing.md:9]:
```suggestion
At its core, Sauce Labs API Testing allows you to use an HTTP client or a proprietary test composer to generate and edit intelligent contract or functional tests. The tests can be automated with any CI/CD tool or using the platform’s scheduler. Functional tests can then be used as load tests or scheduled as uptime monitors.
```

**Review by lysannep — APPROVED** [review]
Looks good to me. Added a couple minor suggestions.

**Review by aoleszyn — APPROVED** [review]
Besides the small typo mentioned by @lysannep , LGTM :)

File: docs/api-testing.md [hunk 3]
```diff
@@ -1,51 +1,78 @@
...
-Looking for our on-premise (legacy) docs? See the [API Fortress documentation](https://docs.saucelabs.com/api-testing/on-prem/quick-start/).
-:::
+
+<div className="box box1 card">
+<div className="container">
+<h2>Contract Testing</h2>
+<p>
+</p>
+<p>
+Use the API Testing platform to generate contract tests from an OpenAPI specification file. When used with Piestry, our mocking tool, you can test both the consumer and the provider.
```
Comment (LoryApiFortress) [inline docs/api-testing.md:20]:
```suggestion
Use the API Testing platform to generate contract tests from an OpenAPI specification file. When used with Piestry, our mocking tool, you can test both the consumer and the producer side.
```

File: docs/api-testing.md [hunk 4]
```diff
@@ -1,51 +1,78 @@
...
+<ul>
+<li>
+<a href="/api-testing/quickstart">HTTP client</a></li>
+<li>
+<a href="/api-testing/import-postman-collection">Postman collection (in the HTTP client)</a></li>
+<li>
+<a href="/api-testing/build-from-spec">OpenAPI spec file or HAR file</a>
+</li>
+<li>
+<a href="/api-testing/import-postman-collection/#importing-a-har-file-from-an-rdc-job">HAR files captured during real device testing</a>
```
Comment (LoryApiFortress) [inline docs/api-testing.md:46]: link above is no longer valid, now we have both RDC and VDC. Replace with:
<a href="/api-testing/import-postman-collection/#importing-a-har-file-rdc-and-vdc">Import from HAR file (including HAR files from RDC and VDC)</a>

File: docs/api-testing.md (hunk 1 above)
Comment (LoryApiFortress) [inline docs/api-testing.md:73]:
```suggestion
Use Sauce Connect Proxy to test APIs behind a firewall. This launches a secure tunnel between the Sauce Labs API Testing platform and your APIs. See <a href="/api-testing/sauce-connect/">Sauce Connect Proxy</a> for more information.
```

**Review by LoryApiFortress — CHANGES_REQUESTED** [review]
The Load testing part is missing:
```suggestion
Load Testing
The platform allows for the creation and execution of load tests using existing functional tests. So once you’ve generated or created a functional test, you can create the load test using these tests.

<a href="/api-testing/load-testing/">Load Testing with API Testing</a>
```

**Comment by lauramthomas** [comment]
@LoryApiFortress Please re-review and approve

**Review by LoryApiFortress — APPROVED** [review]

**Pull request merged**